pom.xml

```diff
@@ -79,19 +79,26 @@
                     <groupId>io.netty</groupId>
                     <artifactId>netty-all</artifactId>
                 </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-common</artifactId>
-            <version>3.3.2</version>
-            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.hadoop</groupId>
+                    <artifactId>hadoop-client</artifactId>
+                </exclusion>
```

Comment:
Is it necessary to exclude this here when you include it later with the explicit version?

Reply:
I put it there because of the vulnerabilities. But just checked removing it from exclusions seems to be okay since we added later versions below.